doc/sphinx/reference/thermo/phase-thermo.md

```diff
@@ -0,0 +1,163 @@
+# Phase Thermodynamic Models
```

Comment:
@ischoegl and I discussed the options for where to document the phase models in https://github.com/Cantera/enhancements/issues/178, with his preference being to leave the detailed documentation for these classes in the header files / Doxygen documentation. This page was my attempt to at least briefly describe all the models available and link to the corresponding Doxygen docs for additional details.

Reply:
Indeed, there was a long discussion on this. My preference was to strengthen the existing (and voluminous!) doxygen documentation. It is my opinion that pointing users in the right direction (while keeping documentation close to the source code) is ultimately more efficient when it comes to long-term code maintenance.

Reply:
Wow, yes, that was a long discussion.  :) 

So do the links to the source code documentation render here on the 'Science' website, then (I am just viewing these files in the GitHub web interface, so couldn't fully tell). Then yes, I agree that this is more than sufficient and avoids having to update in multiple places whenever new content is added or updates are made.

Reply:
Here's a "live" version of the site built from the current PR (more or less): https://testing.cantera.org/pr1647/reference/thermo/index.html. For each phase model, there is a link to (a) the section of the YAML documentation that explains the model-specific parameters and (b) the C++ documentation, where you can find the most detailed description of the model that we currently have. This varies from quite thorough (https://testing.cantera.org/pr1647/cxx/db/d25/classCantera_1_1CoverageDependentSurfPhase.html) to basically nothing (https://testing.cantera.org/pr1647/cxx/d6/d29/classCantera_1_1RedlichKwongMFTP.html).

Reply:
@speth ... thanks for your work on this as well as for providing the link to rendered output. I'm generally :+1: on this (of course!), but noticed that some links to C++ docs are missing: e.g. [here](https://testing.cantera.org/pr1647/reference/thermo/species-thermo.html#peng-robinson-equation-of-state) and [here](https://testing.cantera.org/pr1647/reference/thermo/species-thermo.html#redlich-kwong-equation-of-state) - although I didn't check everything. 

Apart from this, there are two minor nit picks: it would be great if C++ class names could be typeset in fixed width fonts. Also, in some instances, 'implemented by' doesn't mention that links refer to C++ classes (e.g. [here](https://testing.cantera.org/pr1647/reference/thermo/phase-thermo.html#models-for-surfaces-and-interfaces) and other subsections on that page), which I believe is important for establishing the proper context.

Reply:
> it would be great if C++ class names could be typeset in fixed width fonts

I agree, but I think this would require a modification to the `sphinxcontrib-doxylink` plugin, which is how these links are generated. The current formatting is basically what we've always had for these links (see e.g. https://cantera.org/documentation/docs-2.4/sphinx/html/cython/thermo.html#constantcp).

> Also, in some instances, 'implemented by' doesn't mention that links refer to C++ classes (e.g. [here](https://testing.cantera.org/pr1647/reference/thermo/phase-thermo.html#models-for-surfaces-and-interfaces) and other subsections on that page), which I believe is important for establishing the proper context.

At the top of the page, the introduction says "with links to the documentation for their YAML input parameters and the documentation for the C++ classes which implement these models.". I think it would be redundant to mention it every time, given that _all_ thermo models in Cantera are implemented in C++.

Reply:
> > it would be great if C++ class names could be typeset in fixed width fonts
> 
> I agree, but I think this would require a modification to the `sphinxcontrib-doxylink` plugin, which is how these links are generated. The current formatting is basically what we've always had for these links (see e.g. https://cantera.org/documentation/docs-2.4/sphinx/html/cython/thermo.html#constantcp).

Ha! I now recall that there was a limitation that was more or less impossible to circumnavigate - there was some earlier discussion on this. Regarding the other nit, I know it's redundant, but then some may not read what is listed on top (I didn't 😆) - I'm :+1: either way.